### PR TITLE
Compression is a WHATWG standard now

### PR DESCRIPTION
**Pull request by annevk**
***
<a href="https://whatpr.org/compression/60.html" title="Last updated on May 28, 2024, 2:42 PM UTC (9cef4c6)">Preview</a> | <a href="https://whatpr.org/compression/60/34d39f1...9cef4c6.html" title="Last updated on May 28, 2024, 2:42 PM UTC (9cef4c6)">Diff</a>

### CODE REVIEW

**Review by saschanaz — APPROVED** [review]
LGTM even though I don't have a real approval power 😛

File: index.bs [hunk 1]
```diff
@@ -230,5 +222,9 @@ function decompressBlob(blob) {
 }
 </pre>
 
-# Acknowledgments #  {#acknowledgments}
-The editors wish to thank Domenic Denicola and Yutaka Hirano, for their support.
+<h2 class="no-num" id="acknowledgments">Acknowledgments</h2>
+Thanks to Canon Mukai, Domenic Denicola, and Yutaka Hirano, for their support.
```
[the remark this answers is not in the document]
Comment (annevk) [reply index.bs:226]: Thanks, I noted that in the commit message.

**Comment by annevk** [comment]
@ricea obviously you are welcome to tweak the details here. I didn't want to wait another day while a document with a somewhat confusing style sheet was up. Hopefully that's okay.

**Comment by ricea** [comment]
Thank you very much!

**Pull request merged**